src/plugins/telemetryTable/components/table.vue

```diff
@@ -270,7 +315,8 @@ export default {
             scrollW: 0,
             markCounter: 0,
             paused: false,
-            markedRows: []
+            markedRows: [],
+            showingMarkedRowsOnly: false
```

Comment:
This flag can use a better name, following the boolean variable naming convention.